sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs

```diff
@@ -1875,7 +1883,7 @@ private void SetNameFieldsIfNull()
         /// </list>
         /// </remarks>
         public virtual async Task<Response<BlobDownloadResult>> DownloadContentAsync(
-            CancellationToken cancellationToken) =>
+            CancellationToken cancellationToken = default) =>
```

Comment:
It's corresponding sync method `DownloadContent` has optional cancellationToken.